### DISCUSSION TITLE
Small improvements

### DISCUSSION THREAD

**Discussion by DirkDuesentrieb** [45Drives/cockpit-navigator#27] [Ideas]
Thanks for the navigator, it's helpful and allows assigning simple tasks to inexperienced users.
Still I noticed that editing and uploading needed explanation. My ideas:
 Editing: Maybe you can add an "edit" button in the right area (next to the trashcan) 
 Uploading: A "+" button in the left area or a text "Drag files here to upload..." might help

Regards,
Dirk

**Comment by joshuaboud** [comment]
These are some good ideas. Thank you for sharing, I'll look into adding buttons for editing and file upload.

**Comment by joshuaboud** [comment]
I implemented the "edit contents" and "upload" buttons and the new version containing these changes will be released shortly

**Comment by joshuaboud** [comment]
Done with #32

**Reply by DirkDuesentrieb** [reply]
Works great, thanks!
